README.md

```diff
@@ -375,6 +375,19 @@ case we recommend using this test instead.
 ```
 
 ---
+#### json_paths_not_null ([source](macros/schema_tests/json_paths_not_null.sql))
+This test confirms that all rows in a model have json values for the paths specified in the column tested against.
+
+**Usage:**
+```yaml
+- name: subscriptions
+  columns:
+    - name: settings_json
+      tests:
+        - dbt_utils.json_path_not_null:
+            paths: ['preferences.email_type', 'preferences.email_address']
```

Comment:
I think my preference would be to have this macro work like so:
```suggestion
        - dbt_utils.json_path_not_null:
            path: preferences.email_type
        - dbt_utils.json_path_not_null:
            path: preferences.email_address
```
That way, it will be easier to debug when this fails, as you can figure out exactly which path failed.

This will require changes to the macro code (see below).

FWIW, here's the way to write this on different warehouses:
```
-- snowflake
settings_json:preferences.email_address
settings_json:preferences:email_address

-- bigquery
settings_json.preferences.email_address

-- redshift
json_extract_path_text(settings_json, 'preferences', 'email_address'),

-- postgres
settings_json -> 'preferences' ->> 'email_address',
```

One side benefit of only testing one path at a time, is that we could then provide each path as an array:
```
        - dbt_utils.json_path_not_null:
            path: [preferences, email_type]
        - dbt_utils.json_path_not_null:
            path: [preferences, email_address]
```
(or at least do so more cleanly)

Having the keys in an array will make it easier for someone to (later) add multi-warehouse compatibility!